### ISSUE TITLE
Nutrition: add reference from log to meal

### ISSUE THREAD

**Issue by rolandgeider** [wger-project/wger#819]
## Use case

At the moment the different logs only have a foreign key to the plan and it's not possible to know to which meal they belong (other than the time, but that's error prone). This would also allow to show some statistics about the different meals, etc.

## Proposal

Add an optional foreign key from Log to Meal. When a meal is "quick logged" the reference can be set automatically, when the user logs an individual ingredient, present an optional dropdown

**Comment by Alig1493** [comment]
I can make this change on the backend if it's possible to contribute here.

**Comment by rolandgeider** [comment]
Hi! of course, go ahead

**Comment by Alig1493** [comment]
Would I need to update the behaviour of the `_logMealPlan` function to accommodate the new changes?

**Comment by rolandgeider** [comment]
yeah, _logMealPlan is a shortcut function to log the whole meal so that would need the new reference. On the LogCreateView we'd need a dropdown so the user can select a meal (or none)

**Comment by Alig1493** [comment]
Do i also have to send the list of meal ids via context while adding a `ModelMultipleChoiceField` in the forms?

**Comment by rolandgeider** [comment]
I think it's enough if you specify the queryset in the form
